### PR TITLE
Upgrade to Unity5

### PR DESCRIPTION
**Pull request by randomPoison**
:D

### CODE REVIEW

**Comment by randomPoison** [comment]
Before we do the final upgrade we should go through and cache off any of the results of anynew `GetComponent()` calls. Otherwise there weren't any other major changes made in upgrading.

File: Assets/Scripts/Player/PlayerCamera.cs [hunk 1]
```diff
@@ -42,11 +42,17 @@ public class PlayerCamera : ActorComponent
...
 	[SerializeField] LayerMask _collisionLayer = 0;
 
+	private Rigidbody _rigidbody;
+
 	public override void Awake()
 	{
 		base.Awake();
+
+		Cursor.lockState = CursorLockMode.Locked;
+		Cursor.visible = false;
```
Comment (AirDePaul) [inline Assets/Scripts/Player/PlayerCamera.cs]: THANK YOU

Comment (brickman1444) [inline Assets/Scripts/Player/PlayerCamera.cs:52]: THANK YOU

**Comment by brickman1444** [comment]
Done. Everything looks good.

**Pull request merged**